client/layout/guided-tours/tours/marketing-connections-tour/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import { ButtonRow, makeTour, Quit, Link, Step, Tour } from 'layout/guided-tours/config-elements';
+import { localizeUrl } from 'lib/i18n-utils';
+
+export const marketingConnectionsTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			arrow="right-middle"
+			name="init"
+			placement="beside"
+			style={ {
+				marginLeft: '-24px',
+			} }
+			target={ '.sharing-service.not-connected .button.is-compact' }
+		>
+			{ ( { translate } ) => (
+				<>
+					<p>
+						{ translate(
+							"Select the service which you'd like to connect. " +
+								'Whenever you publish a new post, your social media followers will receive an update.'
+						) }
+					</p>
+					<ButtonRow>
+						<Link href={ localizeUrl( 'https://wordpress.com/support/publicize/' ) }>
```

Comment:
Optional: What if we use an inline support link here in order to keep the user within Calypso?